src/services/widgets/widgets.spec.js

```diff
@@ -82,53 +81,50 @@ describe('<> ImpacWidgetsSvc', function () {
         }
       };
 
-      dashboard.widgets[0].settings = [angular.copy(settings)];
-      dashboard.widgets[1].settings = [angular.copy(anotherSettings)];
-      dashboard.widgets[2] = angular.copy(dashboard.widgets[0]);
+      dashboard.widgets[0].settings = [angular.copy(aSetting)];
+      dashboard.widgets[1].settings = [angular.copy(anotherSetting)];
+      dashboard.widgets.push({id: 3, name: 'w-3', metadata: {organization_ids: ['org-1']}})
+      dashboard.widgets[2].settings = angular.copy(dashboard.widgets[0].settings);
 
-      paramName = _.keys(settings.toMetadata())[0];
+      settingKey = _.keys(aSetting.toMetadata())[0];
 
-      spyOn(ImpacDashboardsSvc, 'update').and.callFake(function (id, settings) {
+      spyOn(ImpacDashboardsSvc, 'update').and.callFake(function (id, data) {
         updatedDashboard = angular.copy(dashboard);
-        updatedDashboard[paramName] = settings[paramName];
+        updatedDashboard.metadata[settingKey] = data.metadata[settingKey];
         return $q.resolve(updatedDashboard);
       });
 
-      spyOn(svc, 'update').and.callFake(function (wgt, metadata) {
-        wgt.metadata[paramName] = _.cloneDeep(metadata[paramName]);
-        wgt.isLoading = true;
-        return $q.resolve(wgt);
+      spyOn(svc, 'update').and.callFake(function (wgt, data) {
+        return $q.resolve(wgt.id);
       });
 
       spyOn(svc, 'show').and.callFake(function (updatedWidget) {
         return $q.resolve(updatedWidget);
       });
-      svc.updateAllSameWidgets(dashboard, settings);
-    });
 
-    describe('on _self.load success', function () {
-      beforeEach(function() {
-        spyOn(ImpacDashboardsSvc, 'getCurrentDashboard').and.returnValue(dashboard);
-        $rootScope.$apply();
-      });
+      spyOn(ImpacDashboardsSvc, 'getCurrentDashboard').and.returnValue(dashboard);
 
-      it('expect dashboard setting "reach" will be "dashboard"', function () {
-        expect(updatedDashboard[paramName].reach).toBe("dashboard");
-      });
+      svc.updateAllSimilarWidgets(dashboard, aSetting);
+      $rootScope.$apply();
+    });
 
-      it('updates all the similar widgets', function () {
-        expect(svc.update).toHaveBeenCalledWith(dashboard.widgets[0], {metadata: dashboard.widgets[0].metadata});
-        expect(svc.update).toHaveBeenCalledWith(dashboard.widgets[2], {metadata: dashboard.widgets[2].metadata});
-      });
+    it('updates the dashboard\'s metadata', function () { 
```

Comment:
convention across the project: "it" labels are indicative sentences:
`it does that...`